tests/op_datasets/test_daily_address_summary/test_daily_address_summary.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+from op_datasets.logic.transforms.daily_address_summary import (
+    daily_address_summary,
+    CONDITIONS,
+)
+import polars as pl
+from decimal import Decimal
+
+
```

Comment:
is it possible to also write tests that ensure `dt, address` combinations are unique?